extensions/blocks/map/map.php

```diff
@@ -25,7 +25,98 @@
 function jetpack_map_block_load_assets( $attr, $content ) {
 	$api_key = Jetpack_Options::get_option( 'mapbox_api_key' );
 
+	if ( class_exists( 'Jetpack_AMP_Support' ) && Jetpack_AMP_Support::is_amp_request() ) {
+		static $map_block_counter = [];
```

Comment:
While we can use short array syntax in PHP 5.6+, WordPress core opted against recommending their use in their standards. I would consequently recommend that we follow Core's lead. You can [find out more here](https://make.wordpress.org/core/2019/07/12/php-coding-standards-changes/).

```suggestion
		static $map_block_counter = array();
```

Reply:
Fixed: https://github.com/Automattic/jetpack/pull/13405/commits/9c4d7946ff7e8265e5505a9e0b4ee4fab4d6a2fb